src/Umbraco.Core/Services/IDataTypeService.cs

```diff
@@ -14,6 +14,7 @@ public interface IDataTypeService : IService
     /// <param name="id"></param>
     /// <returns></returns>
     IReadOnlyDictionary<Udi, IEnumerable<string>> GetReferences(int id);
+    IReadOnlyDictionary<Udi, IEnumerable<string>> GetListViewReferences(int id);
```

Comment:
Same as the other interface, this is also a breaking change, which can be fixed in the same way.